sdk/nodejs/cmd/pulumi-language-nodejs/testdata/sdks/simple-1.0.0/package.json

```diff
@@ -5,7 +5,7 @@
         "build": "tsc"
     },
     "dependencies": {
-        "@pulumi/pulumi": "^3.42.0"
+        "@pulumi/pulumi": "../../artifacts/pulumi-pulumi-1.0.0.tgz"
```

Comment:
This is the important fix here. This SDK now links against the packed SDK from the current branch being tested.